### DISCUSSION TITLE
install of raster-vision

### DISCUSSION THREAD

**Discussion by georgeper2001** [azavea/raster-vision#2151] [General]
I am trying to install raster-vision 0.20. I am at the computer's "xxxx@yyyy:~/source$" level, running ubuntu.
I have used the command "pip install rastervision==0.20", which gets an error when trying to install the gdal part.
I think I confirmed this by installing the seperate commands:
"> pip install rastervision_pipeline==0.20
> pip install rastervision_aws_s3==0.20
> pip install rastervision_aws_batch==0.20
> pip install rastervision_core==0.20
> pip install rastervision_pytorch_learner==0.20
> pip install rastervision_pytorch_backend==0.20
> pip install rastervision_gdal_vsi==0.20"
and it erroring when it gets to the last command, "pip install rastervision_gdal_vsi==0.20".

I have to main questions. 1) where does it install to?  and 2) how do I resolve the error?

**Comment by AdeelH** [comment]
You might need to install gdal via conda. As done in the [Dockerfile](https://github.com/azavea/raster-vision/blob/4da111894d93ece46b157b08ffb3e0561d58e4a1/Dockerfile#L38).
